printer_mods/README.md

```diff
@@ -33,6 +33,7 @@ like so:
 |Dan88 |[V0 one-part Skirts](./Dan88/V0skirt) |One part Skirts for the V0 |:heavy_check_mark: |:x: |:x: |
 |DorjeDorf | [Duet 3 6HC Din Bracket](./DorjeDorf/Duet_3_6HC_Din_Bracket) | V2.4 Duet 3 6HC Din Bracket | :x: | :x: | :heavy_check_mark: | 
 | |[Duet 3 3HC Din Bracket](./DorjeDorf/Duet_3_3HC_Din_Bracket) |  V2.4 Duet 3 3HC Din Bracket | :x: | :x: | :heavy_check_mark: | 
+|doubletrouble |[V0 Skirt Mod](./doubletrouble023/V0_Skirt_Mod) |V0 Skirt Mod |:x: |:x: |:heavy_check_mark: |
```

Comment:
Link is broken, the directory you used is `doubletrouble`, not `doubletrouble023`.  Also, you need a checkmark under the V0 column, not the V2.4 column.

Reply:
Oops, sorry. Fixed now